scss/_layout.scss

```diff
@@ -1,51 +1,39 @@
-@import '../grid/helpers';
+@import 'grid';
 
-.topper {
+@mixin oTopper {
 	//Basic topper uses the same grid columns as an article, so things can line up.
-	@include articleGrid();
+	@include oTopperArticleGrid();
 	margin-bottom: 20px;
 
 	@include oGridRespondTo(L) {
-		-ms-grid-rows: 20px min-content 60px min-content 20px;
 		grid-template-rows: 20px min-content 60px min-content 20px;
 	}
 }
 
-.topper--centered {
-	@include articleGridCentered();
-	.topper__read-next {
+@mixin oTopperCentered {
+	@include oTopperArticleGridCentered();
+
+	.o-topper__read-next {
 		display: none;
 	}
 }
 
-.topper--basic {
+@mixin oTopperBasic {
 	@include oGridRespondTo($until: L) {
 		margin-top: 10px;
```

Comment:
Is this top margin going to be suitable for the app? Looks like a response to the nav changing on ft.com. The top margin might be better being added by the client?

Reply:
App has a comparable margin (8px) on the header, i think it's fine